cpu/fe310/include/periph_cpu.h

```diff
@@ -138,6 +138,24 @@ typedef struct {
     i2c_speed_t speed;          /**< I2C speed */
 } i2c_conf_t;
 
+/**
+ * @name    WDT upper and lower bound times in ms
+ * @{
+ */
+#define NWDT_TIME_LOWER_LIMIT           (1)
+#define NWDT_TIME_UPPER_LIMIT           ((UINT32_MAX >> 15)  * MS_PER_SEC + 1)
```

Comment:
Why `(UINT32_MAX >> 15)`?

Reply:
I don't remember, I'll check in the doc.

Reply:
If I can find it, I'll add a comment ;)

Reply:
Looking at the code, it's because internally the `max_time` parameter is multiplied by `RTC_FREQ` which is 32768, e.g. 2¹⁵, and divided by MS_PER_SEC.
This upper limit ensures the internal count value stays without 32bit unsigned int limit.

Reply:
I added a comment mentioning this.